packages/@aws-cdk/cloudformation-include/test/test-templates/bucket-with-parameters.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Name of this file is off (there's only one parameter, so should be `bucket-with-parameter.json`).